templates/profile.html

```diff
@@ -0,0 +1,29 @@
+{% extends 'layout.html' %}
+
+{% block body %}
+<h1> User History </h1>
```

Comment:
`User History` seems a little inappropriate heading for a profile page.

Reply:
okay I'll change that.